testing/trino-tests/src/test/java/io/trino/tests/TestMockConnector.java

```diff
@@ -286,4 +286,26 @@ public void testCaseInsensitive()
                 .hasMessage("execution by operator is not yet implemented for table function simple_table_function");
         assertUpdate("Create Table Mock.Default.New_Table (C int) With (Integer_Table_Property = 1)");
     }
+
+    @Test
+    public void testQuotedSemantics()
```

Comment:
What this test tests? Do you cover #17 is not yet implemented?

Reply:
I have no intentention of merging this commit. It's just to show that quoted semantics is not respected by other statements. I'll drop it later.

Reply:
Extracted to https://github.com/trinodb/trino/pull/15128 to avoid confusion.